submissions/Nikita_Lemesiev/task_js_dom/scripts.js

```diff
@@ -0,0 +1,100 @@
+const series = [
+    ['popular',
+    'Разберитесь с деньгами',
+    '<img id="main-img" src="images/money-problems.png">',
+    'Финансовые консультанты делятся простыми советами о том, как меньше тратить и больше экономить, чтобы контролировать свои средства и достигать задуманного.'],
```

Comment:
I recommend writing an informative structure. Because someone can update your list and make a mistake in the correct order of the entry. 
And avoid the HTML tags in the data list.
```
const series = [
 {
     "category": "popular",
     "title": "Разберитесь с деньгами",
     "poster": "money-problems.png",
     "description": "Финансовые консультанты делятся простыми советами о том, как меньше тратить и больше экономить, чтобы контролировать свои средства и достигать задуманного."
 },
 .....
];
```